.reaction/graphql-linter-rules/type_fields_sorted_alphabetically.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.TypeFieldsSortedAlphabetically = TypeFieldsSortedAlphabetically;
+
+var _validation_error = require('graphql-schema-linter/lib/validation_error');
+
+function TypeFieldsSortedAlphabetically(context) {
+  return {
+    ObjectTypeDefinition: function ObjectTypeDefinition(node) {
+      const fieldList = (node.fields || []).map((field) => field.name.value);
+      const sortedFieldList = fieldList.slice().sort();
+
+      if (!arraysEqual(fieldList, sortedFieldList)) {
+        context.reportError(
+          new _validation_error.ValidationError(
+            'type-fields-sorted-alphabetically',
+            'The fields of object type `' +
+              node.name.value +
+              '` must be sorted alphabetically.' +
+              ' Expected sorting: ' +
+              sortedFieldList.join(', '),
+            [node]
+          )
+        );
+      }
+    }
+  };
+}
+
+function arraysEqual(a, b) {
```

Comment:
Maybe DRY this up a bit with a helper function/module. (The `arraysEqual` function and the slice/sort bit)

Reply:
Done